internal/reporter/gitlab/reporter.go

```diff
@@ -171,9 +175,46 @@ func (r *reporter) SendPullRequestTriggerResult(configCtrl internal.ConfigContro
 	return nil
 }
 
-func (r *reporter) post(gitlabConfig *s2hv1.ReporterGitlab,
-	repository, commitSHA, labelName, targetURL, description string,
-	commitStatus gitlab.CommitStatus, event internal.EventType) error {
+// SendPullRequestTestRunnerPendingResult implements the reporter SendPullRequestTestRunnerPendingResult function
+func (r *reporter) SendPullRequestTestRunnerPendingResult(configCtrl internal.ConfigController,
+	prTestRunnerRpt *internal.PullRequestTestRunnerPendingReporter) error {
+
+	teamName := prTestRunnerRpt.TeamName
+	gitlabConfig, err := r.getGitlabConfig(teamName, configCtrl)
+	if err != nil {
+		return err
+	}
+
+	projectID, err := r.getGitlabProjectID(configCtrl, teamName, prTestRunnerRpt.BundleName)
+	if err != nil {
+		return err
+	}
+
+	r.overrideGitlabCredential(prTestRunnerRpt.Credential, gitlabConfig)
+
+	commitSHA := prTestRunnerRpt.CommitSHA
+
+	// send pull request log status pending while testrunner pipeline is running
+	err = r.post(gitlabConfig, projectID, commitSHA, LabelNameLogs, "", "",
+		gitlab.CommitStatusPending,
+		internal.PullRequestQueueType,
+	)
+	if err != nil {
+		return err
+	}
+	return nil
```

Comment:
can we just
```
return err
```

or 
return r.post(gitlabConfig, projectID, commitSHA, LabelNameLogs, "", "",
		gitlab.CommitStatusPending,
		internal.PullRequestQueueType,
	)